scripts/travisci_run.sh

```diff
@@ -17,6 +17,9 @@ export BUCK_NUM_THREADS=3
 # Make sure that everything builds in case a library is not covered by a test.
 ./bin/buck build --num-threads=$BUCK_NUM_THREADS src/... test/...
 
+# Validate existence of all test rules.
+./bin/buck test --dry-run --all
+
 # Until we get https://github.com/facebook/buck/pull/479, we cannot run the
 # Go tests in Travis
 ./bin/buck test --num-threads=$BUCK_NUM_THREADS
```

Comment:
I think you should be able to change this to `./bin/buck test --num-threads=$BUCK_NUM_THREADS //...` to have it resolve all the `tests` attributes.